### DISCUSSION TITLE
Disk encryption (with LUKS)

### DISCUSSION THREAD

**Discussion by cor** [nix-community/raspberry-pi-nix#33] [Ideas]
Is this setup compatible with LUKS disk encryption solutions such as https://github.com/ViRb3/pi-encrypted-boot-ssh ? If not, how can we apply boot/disk encryption to this setup?

**Comment by Nebucatnetzer** [comment]
I would like to know this as well.
A similar question is how to change the filesystem.

Currently I use my Pis with UEFI and formatted the SD card to use LUKS with F2FS but it would be great if I could preflash the SD cards again.
